nixos/modules/programs/clash-verge.nix

```diff
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.programs.clash-verge;
+
+  clash-verge-package = pkgs.clash-verge.overrideAttrs (finalAttrs: previousAttrs: {
+    installPhase = ''
+      runHook preInstall
+      mv usr $out
+      mv $out/bin/clash-meta $out/bin/clash-meta-origin
+      mv $out/bin/clash $out/bin/clash-origin
+      ln -s /run/wrappers/bin/clash-meta $out/bin/clash-meta
+      ln -s /run/wrappers/bin/clash $out/bin/clash
+      runHook postInstall
+    '';
+  });
```

Comment:
Don't do this, `/run/wrappers/bin` is ordered before `/run/current-system/sw/bin` in PATH, thus the wrapper automatically takes precedence.

Reply:
Because `clash-verge` use the relative path to run clash core instead of using $PATH environment, I have to link to wrapped clash core back to the `bin/` folder to make `clash-verge` use warped clash core

Reply:
Then please patch clash-verge to use vore from PATH, /run/wrappers is mostly an implementation detail and is not intended to be used as such.

Reply:
It was my fault, maybe i can write a module for this.